### PR TITLE
Tanstack Query. Subprojects

### PR DESCRIPTION
**Pull request by maximgolubtsov1201**

### CODE REVIEW

**Review by turansky — CHANGES_REQUESTED** [review]
Alphabetic order required

File: kotlin-wrappers-bom/build.gradle.kts [hunk 1]
```diff
@@ -35,8 +35,10 @@ dependencies {
         api(project(":kotlin-styled-next"))
         api(project(":kotlin-tanstack-react-table"))
         api(project(":kotlin-tanstack-react-virtual"))
+        api(project(":kotlin-tanstack-react-query"))
         api(project(":kotlin-tanstack-table-core"))
         api(project(":kotlin-tanstack-virtual-core"))
+        api(project(":kotlin-tanstack-react-core"))
```
Comment (turansky) [inline kotlin-wrappers-bom/build.gradle.kts:41]: `kotlin-tanstack-query-core`?

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-wrappers-bom/build.gradle.kts (hunk 1 above)
Comment (maximgolubtsov1201) [reply kotlin-wrappers-bom/build.gradle.kts:41]: Fixed typo

**Review by turansky — APPROVED** [review]

**Comment by turansky** [comment]
Thank you!

**Pull request merged**